setup.py

```diff
@@ -12,31 +12,19 @@
 
 
 import sys
-import os
-from setuptools import setup, find_packages, Command
+from setuptools import setup, find_packages
 
 NAME = "docusign_esign"
-VERSION = "2.2.0"
+VERSION = "2.3.0rc1"
 # To install the library, run the following
 #
 # python setup.py install
 #
 # prerequisite: setuptools
 # http://pypi.python.org/pypi/setuptools
 
-
 REQUIRES = ["urllib3 >= 1.15", "six >= 1.8.0", "certifi >= 14.05.14", "python-dateutil >= 2.5.3", "setuptools >= 21.0.0", "PyJWT>=1.7.1", "cryptography>=2.5", "nose>=1.3.7"]
 
-class CleanCommand(Command):
```

Comment:
@harsharahul what are these changes? Don't we need this anymore?

Reply:
yup... @mmallis87  had added these changes along with updates to swagger-codegen-ignore file as these were not necessary. Brought in those changes.

Reply:
did you run the tests locally?

Reply:
yup, did and passed.